packages/atomic/src/components/facets/atomic-date-facet/atomic-date-facet.tsx

```diff
@@ -69,6 +71,10 @@ export class AtomicDateFacet implements InitializableComponent, BaseFacetState {
    * Whether or not the index should automatically generate options for the facet
    */
   @Prop() public generateAutomaticRanges = true;
+  /**
+   * The format that the date will be displayed in. See https://day.js.org/docs/en/display/format for formatting details.
+   */
+  @Prop() public dateFormat = 'DD/MM/YYYY';
```

Comment:
When we have default prop value like that, it's important that they appear in the documentation.

Reply:
The stencil doc gen should add it by default. I'll re-run the doc generation.

Reply:
In atomic-result-date it's `D/M/YYYY`, not super crucial which is which but our formats should be consistant if possible